rtrlib/rtr/packets.c

```diff
@@ -154,6 +155,18 @@ struct pdu_reset_query {
 	uint32_t len;
 };
 
+struct pdu_aspa {
+	uint8_t ver;
+	uint8_t type;
+	uint16_t zero;
+	uint32_t len;
+	uint8_t flags;
+	uint8_t afi_flags;
+	uint16_t provider_count;
+	uint32_t customer_asn;
+	uint32_t provider_asns[];
```

Comment:
this directly translates to `uint32_t* provider_asns` and I think this is a tiny bit better, because of uniformity.

Reply:
leaving it as it is would be consistent with other pdu types in `packets.c`

Reply:
fair